src/main/java/org/example/homework10/Task2/Employee.java

```diff
@@ -0,0 +1,19 @@
+package org.example.homework10.Task2;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+record Employee(String firstName, String lastName, String hireDate) {
+    // Provide public getter methods for firstName, lastName, and hireDate
```

Comment:
remove getters. Record has it by default, but without `get`. Ypu can access every field as it is